### ISSUE TITLE
Add install/build options to requirements.txt

### ISSUE THREAD

**Issue by marcovc** [pfultz2/cget#74]
Is there a way to add install options to the requirements.txt file, such as:

package1 -DOption1=1
package2 -DOption2=2

?

Thanks

**Comment by pfultz2** [comment]
Yes, this is already supported, see [here](https://cget.readthedocs.io/en/latest/src/requirements.html#cmdoption-requirements-txt-d).

**Comment by pfultz2** [comment]
You cant set those flags because they are usually set by the toolchain. Although there should be a way to add additional flags for some projects.

However, for your example, you can add `-DCMAKE_POSITION_INDEPENDENT_CODE=On` to add the `-fPIC` flag.

**Comment by marcovc** [comment]
I see. But then why does it work if set those flags when I call cget?

I.e., this:

cget install -Detc

works in this case.

**Comment by pfultz2** [comment]
> I see. But then why does it work if set those flags when I call cget?

Variables like `CMAKE_CXX_FLAGS` are set by cget's internal cmake toolchain file, so it will override the flags set by -`Detc`.
